### PR TITLE
Add docker environment

### PR DESCRIPTION
**Pull request by SimonBaeumer**
This pull request adds docker support. 

**Configuration:** 

There is no need to configure something at the moment. You are able to copy the .psh.yml.dist to your plugin root and use it!

### CODE REVIEW

File: docker/templates/Dockerfile.tpl [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+     ant \
+     unzip \
+     git
+
+RUN docker-php-ext-configure gd --with-jpeg-dir=/usr/lib \
+  && docker-php-ext-install pdo pdo_mysql mbstring gd zip
+
+RUN a2enmod rewrite
+
+WORKDIR /var/www/html/engine/Shopware/Plugins/Local/Frontend/SwagBundle
```
Comment (JanPietrzyk) [inline docker/templates/Dockerfile.tpl]: hey

File: docker/templates/Dockerfile.tpl [hunk 2]
```diff
@@ -0,0 +1,19 @@
+FROM php:__PHP_VERSION__-apache
```
Comment (JanPietrzyk) [inline docker/templates/Dockerfile.tpl:1]: user zeuch fehlt: sowas wie das hier:

```
DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"

USERID=$(id -u)
GROUP=$(id -g)


cat > $DIR/php7/createuser.sh <<DELIM
#!/usr/bin/env bash

groupadd -g ${GROUP} app-shell
useradd -s /bin/bash -m -u ${USERID} -g ${GROUP} app-shell
mkdir -p /home/app-shell/.ssh
chown -R app-shell:app-shell /home/app-shell
echo -e "app-shell\napp-shell\n" | passwd app-shell
echo 'app-shell  ALL=(ALL:ALL) NOPASSWD: ALL' >> /etc/sudoers
DELIM
```

File: docker/templates/Dockerfile.tpl (hunk 1 above)
Comment (SimonBaeumer) [reply docker/templates/Dockerfile.tpl]: done

File: docker/templates/Dockerfile.tpl (hunk 2 above)
Comment (SimonBaeumer) [reply docker/templates/Dockerfile.tpl:1]: Ist implementiert

**Pull request merged**